src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts

```diff
@@ -224,32 +231,34 @@ describe('HmctsGlobalHeaderComponent', () => {
       })
     ).subscribe(items => {
       expect(items).toEqual([component.items[0]]);
-      done();
     });
   });
 
-  it('filters out menu items for which not all features are enabled', (done) => {
+  it('filters out menu items for which not all features are enabled', async () => {
     component.items = [{
       align: 'right',
       text: '1',
       href: '',
       active: false,
-      flags: ['enabledFlag']
+      flags: ['enabledFlag'],
+      roles: ['roleA']
     },
     {
       align: null,
       text: '2',
       href: '',
-      active: false
+      active: false,
+      roles: ['roleB']
     },
     {
       align: 'right',
       text: '3',
       href: '',
       active: false,
-      flags: ['enabledFlag', 'disabledFlag']
+      flags: ['enabledFlag', 'disabledFlag'],
+      roles: ['roleC']
     }];
-    component.ngOnChanges(changesMock);
+    await component.ngOnChanges(changesMock);
```

Comment:
@johnbenjamin-hmcts - see previous comment on `await`.

Reply:
Thank you @Daniel-Lam - I have now reverted the changes.